.gitignore

```diff
@@ -8,3 +8,4 @@ resources/overlays/*
 .idea
 btsync-media.env 
```

Comment:
delete

Reply:
as said I'd like to keep the files as close to the original ones to avoid rebasing conflicts